aioircd.py

```diff
@@ -285,7 +284,23 @@ async def JOIN(self, args):
 
     @command
     async def PRIVMSG(self, args):
-        pass
+        if not args or args[0] == "":
+            raise ErrNoRecipient("PRIVMSG")
+        user,*mess = args
+        message = " ".join(mess)
```

Comment:
Maybe an abuse of the python uncap ? I'm not sure it would be crystal clear on demo
```
user = args[0]
msg = " ".join(args[1:])
```